cli/command/context/list_test.go

```diff
@@ -30,6 +31,25 @@ func TestList(t *testing.T) {
 	golden.Assert(t, cli.OutBuffer().String(), "list.golden")
 }
 
+func TestListJSON(t *testing.T) {
```

Comment:
nit: shall we also add a small test for the other two (unexpected) values of ContextType for the old cloud integrations?

Reply:
This is done; I'll leave a comment below